common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -948,6 +948,13 @@ def make_course_key(self, org, course, run):
         """
         return CourseLocator(org, course, run)
 
+    def make_course_usage_key(self, course_key):
+        """
+        Return a valid :class:`~opaque_keys.edx.keys.UsageKey` for this modulestore
+        that matches the supplied course_key.
+        """
+        return BlockUsageLocator(course_key, 'course', 'course')
```

Comment:
The block cache framework needs this in order to determine the root usage key for the course.
Aren't library usage keys outside the scope of this method?  Should I rename this method to make it more clear?
I don't know how CCX works.  Would this not work for CCX courses?